tabs/mission_control.js

```diff
@@ -189,6 +381,7 @@ TABS.mission_control.initialize = function (callback) {
         });
 
         iconFeature.setStyle(getPointIcon());
+        //iconFeature.setGeometryName('Waypoint');
```

Comment:
@CSMachineCo please remove if not used

Reply:
Ok.  Removed that and added marker for RTH position.